ProjectSource/src/main/java/com/kayako/sdk/base/requester/GetListRequester.java

```diff
@@ -0,0 +1,57 @@
+package com.kayako.sdk.base.requester;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class GetListRequester extends GetRequestProperty {
+
+  private final String endpoint;
+  private final String argId;
+
+  private static final String ARG_OFFSET = "offset";
+  private static final String ARG_LIMIT = "limit";
+
+  private long mId;
+  private String mHelpCenterUrl;
```

Comment:
❗️ This snippet of code contains a **blank**  instance variable assigned in the constructor, but not `final`. Consider marking this instance variable as `final`. So, it should become a **blank final** instance variable. This way the code avoids intentionally/accidentally re-assignment of this instance variable and sustain the best OO practices.  

**[More details](https://confluence.devfactory.com/x/lgAxF)**

## The solution in this case

```Java
private final String mHelpCenterUrl;
```

 

<a id="0a63d640-50c3-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />